src/test/java/com/metamx/emitter/core/LoggingEmitterConfigTest.java

```diff
@@ -17,41 +17,44 @@
 package com.metamx.emitter.core;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import junit.framework.Assert;
+import org.junit.Assert;
 import org.junit.Test;
 
-import java.util.Properties;
-
 public class LoggingEmitterConfigTest
 {
   @Test
-  public void testDefaults()
+  public void testDefaults() throws Exception
   {
-    final Properties props = new Properties();
     final ObjectMapper objectMapper = new ObjectMapper();
-    final LoggingEmitterConfig config = objectMapper.convertValue(
-        Emitters.makeLoggingMap(props),
+    final LoggingEmitterConfig config = objectMapper.readValue(
+        "{}",
         LoggingEmitterConfig.class
     );
 
     Assert.assertEquals(LoggingEmitter.class.getName(), config.getLoggerClass());
-    Assert.assertEquals("debug", config.getLogLevel());
+    Assert.assertEquals("info", config.getLogLevel());
```

Comment:
because earlier, Emitters.makeLoggingMap(props) was used which changes the default (that is there in LoggingEmitterConfig) to "debug".